GitHubLabels.ps1

```diff
@@ -565,6 +628,121 @@ function Set-GitHubLabel
     }
 }
 
+function Add-GitHubLabel
+{
+<#
+    .DESCRIPTION
+        Adds a label to an issue in the given GitHub repository.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER Issue
+        Issue number to add the label to.
+
+    .PARAMETER LabelName
+        Array of label names to add to the issue
+
+    .PARAMETER Replace
```

Comment:
From an internal implementation perspective, I think this is fine.  I'm not sure I buy into this though from an exported API design perspective.  I think it would generally be frowned-upon from a PowerShell naming perspective to combine both "Add" and "Set"/Replace functionality within the same exported function.